src/serac/numerics/equation_solver.hpp

```diff
@@ -116,49 +108,54 @@ class EquationSolver : public mfem::Solver {
                                                                const NonlinearSolverOptions& nonlin_options);
 
   /**
-   * @brief A wrapper class for combining a nonlinear solver with a SuperLU direct solver
+   * @brief A wrapper class for using the MFEM super LU solver with a HypreParMatrix
    */
-  class SuperLUNonlinearOperatorWrapper : public mfem::Operator {
+  class SuperLU : public mfem::Solver {
```

Comment:
I find this class name confusing. At first I thought it was using a class from the SuperLU library not from us. ``SuperLUSolver`` maybe? The previous was very verbose though...

Reply:
Done.